contrib/cmake/build3rdParty/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+
+
+cmake_minimum_required(VERSION 3.13)
```

Comment:
There is no need to re-specify `cmake_minimum_required` and `project` in individual CMLs; it should be done once in the topmost CML

Reply:
Since some people will still be using this one directly, and some perhaps using the one in the root, I think they are both needed or cmake will complain.